README.md

```diff
@@ -396,6 +396,66 @@ There are also 3 networks:
 3. `aws_discover`: network allowing `iot_pub` container to discover Greengrass device. This is
    not required once discovery is complete.
 
+### Automate Robots onboarding to AWS IoT at scale
+New robots must be provisioned in order to discover the GreenGrass server. Provisioning is the process of providing robots with unique identities (e.g., an X.509 certificate and a private key), registering these identities with the AWS IoT endpoint, and adding required IoT permissions so that robots may securely connect and function.
+![fleet_provisioning](images/fleet_provisioning.png)
+
+We going to use AWS IoT fleet provisioning, [Provisioning by claim](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html) to automate and scale the process of registering robot identities with the AWS Cloud and associating the required AWS IoT permissions.
+
+#### Create the Provisioning Template
+For using the fleet provisioning feature of AWS IoT Core, you need to setup an IAM role and a provisioning template in your AWS account.
+1. Create an IAM role that will be needed by a fleet provisioning template
+``` bash
+ROLE=$(aws iam create-role \
+      --role-name provision-by-claim-role \
```

Comment:
Can you set the role name to an environment variable, so the user can modify it?